doc/radosgw/d3n_datacache.rst

```diff
@@ -0,0 +1,93 @@
+====================
+D3N RGW Data caching
+====================
+
+.. contents::
+
+Datacenter-Data-Delivery Network (D3N) uses high-speed storage such as NVMe flash or DRAM to cache
+datasets on the access side.
+Such caching allows big data jobs to use the compute and fast storage resources available on each
+Rados Gateway node at the edge.
+
+Many datacenters include low-cost, centralized storage repositories, called data lakes,
+to store and share terabyte and petabyte-scale datasets.
+By necessity most distributed big-data analytic clusters such as Hadoop and Spark must
+depend on accessing a centrally located data lake that is relatively far away.
+Even with a well-designed datacenter network, cluster-to-data lake bandwidth is typically much less
+than the bandwidth of a solid-state storage located at an edge node.
+
+| D3N improves the performance of big-data jobs running in analysis clusters by speeding up recurring reads from the data lake.
+| The Rados Gateways act as cache servers for the back-end object store (OSDs), storing data locally for reuse.
+
+
+See `MOC D3N (Datacenter-scale Data Delivery Network)`_ and `Red Hat Research D3N Cache for Data Centers`_.
+
+Implementation
+==============
+
+- D3N cache supports both the `S3` and `Swift` object storage interfaces.
+- D3N will cache tail objects (by default it is parts of objects that are larger than 4MB).
+
+Requirements
+------------
+
+- A SSD(/dev/nvme,/dev/pmem,/dev/shm) storage device XFS formatted and mounted, it will be used as the cache backing store.
+  (depending on device performance, multiple RGWs may share a single device but each requires a discrete directory on said device)
+- The `rgw_enable_ops_log` config variable needs be enabled for D3N operation.
```

Comment:
why?  what is the connection with ops-log?

Reply:
was a requirement of the original PR, believe no longer necessary, have a task to re-check, will update

Reply:
checked & not necesary, will remove